cli/src/commands/create/dataset.rs

```diff
@@ -66,6 +76,12 @@ pub fn create(client: &Client, args: &CreateDatasetArgs) -> Result<()> {
                 description: description.as_ref().map(|description| description.as_str()),
                 has_sentiment: *has_sentiment,
                 entity_kinds: &entity_kinds,
+                model_family: model_family
+                    .as_ref()
+                    .map(|model_family| model_family.as_str()),
+                copy_labels_from: copy_labels_from
+                    .as_ref()
+                    .map(|copy_labels_from| copy_labels_from.as_str()),
```

Comment:
I think you're looking for `Option::as_deref`:

```suggestion
                model_family: model_family.as_deref(),
                copy_labels_from: copy_labels_from.as_deref(),
```

(looks like you copied this pattern from other uses of Option in this file - maybe tidy the others up too if you're feeling generous?)